spec/features/course/assessment/submission/worksheet_spec.rb

```diff
@@ -78,6 +78,28 @@
         end
       end
 
+      scenario 'I can edit my existing comment', js: true do
```

Comment:
I'm okay with that. We need to keep this in check though ... travis is taking 16 min currently. We shouldn't let it build for 20 min or more.